frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -175,9 +177,10 @@ return {
         north = "Í",
         northeast = "Í",
         northwest = "Ì",
-        east = "Î",
+        east = "Î", -- I with circumflex (Romanian)
```

Comment:
You're correct. I think that each change for a specific language should add that language to the comment list.

Reply:
You can if you want to try, I suppose. :-)

It's basically something in between [EurKEY](https://eurkey.steffen.bruentjen.eu/) and US International. That gives you Afrikaans, Danish, Dutch, English, Faroese, Finnish, French, German, Icelandic, Irish, Italian, Norwegian, Portuguese, Scottish Gaelic, Spanish, and Swedish as the primary supported languages, with others not necessarily fitting within the confines of the keyboard — especially if it would be at the detriment of one of those.